CHANGELOG.rst

```diff
@@ -7,74 +7,160 @@ v0.2.0 - 2020-xx-xx
+
+What you need to do
+    If you have remote layers deployed in production, you should update the view configuration to contain an "upstream" layer.  This will create a copy of the remote layer data to the local Cortex and keep it in sync.
+
+Removed Pushing Splices
+~~~~~~~~~~~~~~~~~~~~~~~
 
-- Removed splices to cryotank
+What changed
+    The configuration options to enable pushing splices to a cryotank or to another cortex have been removed.
 
-- Removed pushing splices
+Why make the change
+    The archival of splices to a cryotank and the responsibility of a Cortex to "push" changes to another Cortex have long been essentially vestigial.  Additionally, these options required a Cortex reboot to take effect and were not runtime configurable.  The current mechanisms for mirroring and upstream layers allow for a more scalable and dynamic configuration.
 
-- Removed feeds, feed loop
+What you need to do
+    It is unlikely that this change will effect any known deployments.
 
-- Consolidated datamodel APIs
+Removed Monolithic Feed Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- Triggers/cron/queues have creators and each may have admins
+What changed
+    The monolithic configuration option for pulling "feed" data from a Cryotank has been removed.
 
-- Removed #:score tagprop lift
+Why make the change
+    The ability to feed a Cortex directly from a Cryotank represents a very early approach to automate data ingest into a Cortex.  This capability has been superseded by Storm Services which provide a dynamically configurable way to integrate services and data.
 
-- Removed insecure mode
+What you need to do
+    It is unlikely that this change will effect any known deployments.
 
-- snap.model is now accessible at snap.core.model
+Removed Tag Prop Lifting Without Tag
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- no default values anymore
+What changed
+    The ability to lift nodes by the presence of a tag property *without* specifying the tag name has been removed.  Given a tag property of "confidence", the ``#:confidence`` and ``#:confidence>90`` style syntax are no longer valid.  However, lifting by tag property *with* the tag, such as ``#foo.bar:confidence`` and ``#foo.bar:confidence>90`` remain valid.
 
-- snap methods nodesby are now different
+Why make the change
+    The necessary indexing to provide a performant way to lift nodes by the tag property without the tag is too expensive for the analytically dubious use case.
 
-- auth.getUserByName now async (why?)
+What you need to do
+    Any instances of lifting nodes by tag property without the tag will need to be updated to include the tag name.
 
-- user.admin -> user.isAdmin()
+Removed Insecure Mode
+~~~~~~~~~~~~~~~~~~~~~
+
+What changed
+    The "insecure" option in cell.yaml has been removed.
+
+Why make the change
```

Comment:
It is also possible to bootstrap a root password directly via command line / envar / cell.yaml